File: Service/Billing/RMA.Service.Billing.Contracts/Entities/InterestAdjustment.cs
namespace RMA.Service.Billing.Contracts.Entities
{
    public class InterestAdjustment
    {
        public int RoleplayerId { get; set; }
        public int TransactionId { get; set; }
        public decimal AdjustmentAmount { get; set; }
        public Statement Transaction { get; set; }
        public bool IsUpwardAdjustment { get; set; }
        public string FinPayee { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/interest-adjustment.ts
import { Statement } from "../../shared/models/statement";

export class InterestAdjustment {
    transactionId: number;
     adjustmentAmount :number; 
     roleplayerId :number; 
     isUpwardAdjustment :boolean
     transaction: Statement;
     finPayee: string;
  }
  